scripts/list_migrations.py

```diff
@@ -4,9 +4,11 @@
 from __future__ import print_function
 
 import sys
+import warnings
 
 from alembic.script import ScriptDirectory
 
+warnings.simplefilter('error')
```

Comment:
This has angered PEP8.  Needs an extra empty line below.

Otherwise, nice catch!